src/Model/Config/ProjectConfig.php

```diff
@@ -44,9 +43,9 @@ class ProjectConfig
     public const KEY_TASKS = 'tasks';
     public const KEY_UPSTREAMS = 'upstreams';
     public const KEY_COMPOSER_PATH = 'composer_path';
-    public const KEY_DRUSH_MAJOR_VERSION = 'drush_major_version';
-    public const KEY_SYMFONY_CLI = 'symfony_cli';
-    public const KEY_DRUSH_PATCHER_INSTALLED = 'drush_patcher_installed';
+    public const KEY_COMMAND_PREFIX = 'command_prefix';
+    public const KEY_LOCAL_SETTINGS_FILENAME = 'local_settings_filename';
```

Comment:
Same question from above about the local settings file.

Reply:
I had a project where the local settings file was named something differently. This lets you define the secret/local settings file's filename per project.